### ISSUE TITLE
readme - Anpassung für andere Hochschulen

### ISSUE THREAD

**Issue by tobiaskl93** [MrKrisKrisu/Hochschule-Hannover-Pruefungsergebnischecker#4]
Welche Anpassungen an dem Skript sind nötig, damit dies für andere Hochschulen verwendet werden kann?
Reicht es aus, den Part "icms.hs-hannover.de" durch den jeweiligen Link der Hochschule zu ersetzen (bspw. "vorlesungen.tu-bs.de" oder "qis.uni-hannover.de" oder "qisserver.hs-koblenz.de" . . . )?

Ich bekomme dabei immer den Fehler "Link Notenspiegel wurde nicht gefunden."

Es wäre super, wenn Ihr für die Hochschulen, bei denen ihr das Skript getestet habt eine eigenes File hochladen könntet oder die Readme um einen Abschnitt "Anpassungen an die Hochschule" o.ä. erweitern könntet. (:

**Comment by MrKrisKrisu** [comment]
> Welche Anpassungen an dem Skript sind nötig, damit dies für andere Hochschulen verwendet werden kann?
> Reicht es aus, den Part "icms.hs-hannover.de" durch den jeweiligen Link der Hochschule zu ersetzen (bspw. "vorlesungen.tu-bs.de" oder "qis.uni-hannover.de" oder "qisserver.hs-koblenz.de" . . . )?

Ja, in der Regel hat es gereicht, den Link entsprechend anzupassen.

> Ich bekomme dabei immer den Fehler "Link Notenspiegel wurde nicht gefunden."

Es kann sein, dass durch Updates von HISinOne die Linkstruktur mittlerweile anders ist. Da ich aktuell leider keine Möglichkeit habe, mir die Systeme anderer Hochschulen anzuschauen, kann ich dazu aber leider nichts sagen.

> Es wäre super, wenn Ihr für die Hochschulen, bei denen ihr das Skript getestet habt eine eigenes File hochladen könntet oder 
die Readme um einen Abschnitt "Anpassungen an die Hochschule" o.ä. erweitern könntet. (:

Wenn ich die wieder mal Zugriff auf andere Installationen habe, kann ich das gerne machen!
